src/Compilers/VisualBasic/Portable/Declarations/DeclarationModifiers.vb

```diff
@@ -62,21 +53,4 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         InvalidInModule = [Protected] Or [Shared] Or [Default] Or [MustOverride] Or [Overridable] Or [Shadows] Or [Overrides]
         InvalidInInterface = AllAccessibilityModifiers Or [Shared]
     End Enum
-
```

Comment:
pulled out so i could reference DeclarationModifiers without it pulling in lots of other things (like `Accessibility`).